tests/integration/test_events.py

```diff
@@ -230,3 +237,66 @@ def test_put_events_with_target_lambda(self):
         self.events_client.delete_event_bus(
             Name=TEST_EVENT_BUS_NAME
         )
+
+    def test_schedule_expression_event_with_http_endpoint(self):
+        class HttpEndpointListener(ProxyListener):
+            def forward_request(self, method, path, data, headers):
+                event = json.loads(to_str(data))
+                events.append(event)
+                return 200
+
+        topic_name = 'topic-{}'.format(short_uid())
+        rule_name = 'rule-{}'.format(short_uid())
+        target_id = 'target-{}'.format(short_uid())
+        events = []
+
+        local_port = get_free_tcp_port()
+        proxy = start_proxy(local_port, backend_url=None, update_listener=HttpEndpointListener())
+        wait_for_port_open(local_port)
+
+        endpoint = '{}://{}:{}'.format(get_service_protocol(), config.LOCALSTACK_HOSTNAME, local_port)
+
+        topic_arn = self.sns_client.create_topic(Name=topic_name)['TopicArn']
+        self.sns_client.subscribe(TopicArn=topic_arn, Protocol='http', Endpoint=endpoint)
+
+        event = {
+            'env': 'testing'
+        }
+
+        self.events_client.put_rule(
+            Name=rule_name,
+            ScheduleExpression='rate(1 minutes)'
+        )
+
+        self.events_client.put_targets(
+            Rule=rule_name,
+            Targets=[
+                {
+                    'Id': target_id,
+                    'Arn': topic_arn,
+                    'Input': json.dumps(event)
+                }
+            ]
+        )
+
+        def received():
+            self.assertGreaterEqual(len(events), 2)
+            notifications = [event['Message'] for event in events if event['Type'] == 'Notification']
+            self.assertGreaterEqual(len(notifications), 1)
+            return notifications[0]
+
+        notification = retry(received, retries=2, sleep=40)
```

Comment:
This seems like a fairly long waiting time which increases the test duration. One thing we could do is to create the SNS topic and event rules when the test class gets instantiated, like we do here: https://github.com/localstack/localstack/blob/master/tests/integration/test_integration.py#L52-L63 . This way we could potentially save some time, if the event gets triggered in the background and we only leave the assertions in this test method.